js/app.js

```diff
@@ -1,2 +1,89 @@
-$(function () {
+$(document).ready(function () {
+  $('#submit-btn').click(() => {
+
+    refreshSearch()
+
+    const city = $('#city-type').val()
+
+    getWeatherByCity(city)
+
+  })
+
+
+  function getWeatherByCity (city) {
+    axios.get(`http://api.openweathermap.org/data/2.5/weather?q=${city}&APPID=1c739a9713a0919a3024a089c3b027df`)
+    .then((response) => {
+      console.log(response.data)
+      displayCityName(response.data.name)
```

Comment:
@kcycle nice! I like the use of separate functions to display each element. Putting all the display logic into one function would have also adhered to the SRP rule as well.